runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -110,6 +110,7 @@
 	private static final ImmutableList<String> PET_MESSAGES = ImmutableList.of("You have a funny feeling like you're being followed",
 		"You feel something weird sneaking into your backpack",
 		"You have a funny feeling like you would have been followed");
+	private static final Pattern BA_HIGH_GAMBLE_REWARD_PATTERN = Pattern.compile("(.+)!<br>High level gamble count: <col=7f0000>(.+)</col>");
```

Comment:
Now that we're not capturing the reward, we can remove the capturing group and use `.group(1)` to get the gamble count. Alternatively, each of the groups should be given a name using the syntax `(?<name>pattern)` and the matcher changed to use [`Matcher.group(String name)`](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Matcher.html#group-java.lang.String-) to fetch the captured text instead.
```suggestion
	private static final Pattern BA_HIGH_GAMBLE_REWARD_PATTERN = Pattern.compile("(?<reward>.+)!<br>High level gamble count: <col=7f0000>(?<gambleCount>.+)</col>");
```